### PR TITLE
add city to stoppoint

### PR DESCRIPTION
**Pull request by dvdn**
To see city in Models / templates

### CODE REVIEW

File: Services/StopPointManager.php [hunk 1]
```diff
@@ -43,6 +43,17 @@ private function initTitle($externalCoverageId)
         );
     }
 
+    private function initCity($externalCoverageId)
+    {
+        $this->stopPoint->setCity(
+            $this->navitia->getStopPointCity(
+                $externalCoverageId,
+                $this->stopPoint->getExternalId(),
+                ["depth" => 1, "show_codes" => "true"]
```
Comment (datanel) [inline Services/StopPointManager.php]: Timetable requires php >=5.3.2 so [  ] notation is not allowed.
user array() notation

File: Entity/StopPoint.php [hunk 2]
```diff
@@ -30,6 +30,11 @@ class StopPoint extends AbstractEntity
     /**
      * @var Object
      */
+    private $city;
+
+    /**
+     * @var Object
```
Comment (datanel) [inline Entity/StopPoint.php]: object in lowercase

File: Services/Navitia.php [hunk 3]
```diff
@@ -206,6 +206,19 @@ public function getStopPointTitle($coverageId, $stopPointId)
         return ($response->stop_points[0]->name);
     }
 
+    /**
+     * Returns Stop Point City
+     *
+     * @param  String $coverageId
```
Comment (datanel) [inline Services/Navitia.php]: string in lowercase

File: Services/Navitia.php [hunk 4]
```diff
@@ -206,6 +206,19 @@ public function getStopPointTitle($coverageId, $stopPointId)
...
+     * @param  String $stopPointId
```
Comment (datanel) [inline Services/Navitia.php]: string in lowercase

**Comment by dvdn** [comment]
Clean fixes

File: Services/StopPointManager.php [hunk 5]
```diff
@@ -104,6 +115,7 @@ public function getStopPoint($externalStopPointId, $timetable, $externalCoverage
             $this->stopPoint->setTimetable($timetable);
         }
         $this->initTitle($externalCoverageId);
+        $this->initCity($externalCoverageId);
         $this->initStopPointCode($externalCoverageId);
```
Comment (datanel) [inline Services/StopPointManager.php]: $this->initTitle($externalCoverageId); and  $this->initStopPointCode($externalCoverageId); do the same request /v1/coverage/jdr/stop_points/stop_point:TAD:SP:92002/?depth=1&show_codes=true

Is there a way to do the request once?

**Pull request merged**